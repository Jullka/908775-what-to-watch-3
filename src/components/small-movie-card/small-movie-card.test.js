import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card";

const movie =
  {
    title: `Shutter Island`,
    img: `img/shutter-island.jpg`
  };

it(`SmallMovieCard component renders correctly`, () => {
  const tree = renderer
    .create(<SmallMovieCard movie={movie} handleCardOver={() => {}} handleTitleClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
