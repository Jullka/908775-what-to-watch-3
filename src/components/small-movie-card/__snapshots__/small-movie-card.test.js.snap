// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard component renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Shutter Island"
      height="175"
      src="img/shutter-island.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Shutter Island
    </a>
  </h3>
</article>
`;
